apps/prairielearn/src/lib/code-caller/code-caller-container.ts

```diff
@@ -21,14 +20,17 @@ import { makeAwsClientConfig } from '../aws.js';
 import { config } from '../config.js';
 import { deferredPromise } from '../deferred.js';
 
-import { FunctionMissingError } from './code-caller-shared.js';
-
-/** @typedef {typeof CREATED | typeof WAITING | typeof IN_CALL | typeof EXITING | typeof EXITED} CallerState */
-const CREATED = Symbol('CREATED');
-const WAITING = Symbol('WAITING');
-const IN_CALL = Symbol('IN_CALL');
-const EXITING = Symbol('EXITING');
-const EXITED = Symbol('EXITED');
+import {
+  FunctionMissingError,
+  type CallType,
+  CREATED,
+  IN_CALL,
+  EXITING,
+  EXITED,
+  WAITING,
```

Comment:
Nit, IMO we don't need to put these in `code-caller-shared`. It's a happy accident that both callers use sorta-kinda the same state machine, but really these are implementation details of each separate code caller.